pkg/cmd/flags.go

```diff
@@ -79,13 +79,15 @@ var (
 	statementLogFileCompression      string
 	versionFlag                      bool
 	iOWorkerPool                     int
+	randomStringBuffer               int
 
 	concurrency         int
 	mutationConcurrency int
 	readConcurrency     int
 )
 
 func setupFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().IntVarP(&randomStringBuffer, "random-string-buffer-size", "", 100*1024*1024, "Size of the buffer used for random strings")
```

Comment:
Improve this msg to tell the impact: the lower size, the more overlaps. Possibly, this will increase the pressure on the compression (worth to do some tests how much data is compressed we have with low numbers and high ones)